.ebextensions/datadog.config

```diff
@@ -0,0 +1,76 @@
+option_settings:
```

Comment:
possible to add the source in a comment in this file itself pls, so we don't have to refer back to the PR to know where this came from?